.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# style(tree): apply mdformat fixes
+99e3f12c2b09d23369262e4a17912cacab17ee1f
```

Comment:
nah; we just shouldn't do these commits.

Reply:
ack, fine with this.

Reply:
I want to reraise this -- this is a good feature that can help us make changes without clobbering history.  I've dropped the application of clang-format on this PR, but I didn't drop all of linters/fixers and some of those are better ignored. Note that github supports this file and blame should work correctly if you go to the top of this page and click the branch, or this link: https://github.com/aws-nslick/aws-ofi-nccl/tree/qol